File: JAVA/Daily Flash Codes/Week1/Day6/code_Java/prog4.java
/*
*
Program 4 : Write a Program to print First 50 Even Numbers
Output: 2 4 6 . . . 100
*/


class Program {
	public static void main(String[] args)  {
		int num =2;

		for(int i=1; i<= 50; i++){
			System.out.print(num+ "  ");
			num +=2;
		}
		System.out.println();
	}
}

File: C++/Daily Flash/week1/Day6/code_C++/prog4.cpp
/*
*Program 4 : Write a Program to print First 50 Even Numbers
Output: 2 4 6 . . . 100
*
*/

#include <iostream>

int main() {

	std::cout << "First 50 EVEN Numbers: " << std::endl;
	int num =0;

	for(int i=1; i<= 50; i++){
		std::cout << num << " " ;
		num +=2;
	}
	std::cout << std::endl;
	return 0;
}
